reference-adapter/mgramseva-ifix-adapter/src/main/java/org/egov/ifix/service/MgramsevaChallanService.java

```diff
@@ -0,0 +1,10 @@
+package org.egov.ifix.service;
+
+import org.egov.ifix.models.fiscalEvent.FiscalEvent;
+
+import javax.validation.constraints.NotNull;
+
+public interface MgramsevaChallanService {
+    void createChallanInChallanService(String eventType, FiscalEvent fiscalEvent, String mgramsevaTenantId,
```

Comment:
Just name it createChallan. "InChallanService" looks unnecessary.

Reply:
sure